core/src/main/java/com/novoda/landingstrip/animation/FastOutSlowInInterpolator.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.landingstrip.animation;
+
+class FastOutSlowInInterpolator extends LookupTableInterpolator {
```

Comment:
This class has no behaviour, alternative:

`Interpolator fastInSlowOutInterpolator = InterpolatorFactory.getFastOutSlowInInterpolator();`

with

```
class InterpolatorFactory {

    public Interpolator getFastOutSlowInInterpolator() {
       return new LookupTableInterpolator(FAST_IN_SLOW_OUT_VALUES);
    }
 }
```